geode-redis/src/main/java/org/apache/geode/redis/internal/GeodeRedisServer.java

```diff
@@ -93,13 +86,21 @@ public GeodeRedisServer(String bindAddress, int port, InternalCache cache) {
     CommandFunction.register(regionProvider.getDataRegion(), stripedExecutor, redisStats);
     RenameFunction.register(regionProvider.getDataRegion(), stripedExecutor, redisStats);
     RedisCommandFunction.register();
+
     passiveExpirationManager =
         new PassiveExpirationManager(regionProvider.getDataRegion(), redisStats);
-    redisCommandExecutor = LoggingExecutors.newCachedThreadPool("GeodeRedisServer-Command-", true);
-    nettyRedisServer = new NettyRedisServer(() -> cache.getInternalDistributedSystem().getConfig(),
-        regionProvider, pubSub,
-        this::allowUnsupportedCommands, this::shutdown, port, bindAddress, redisStats,
-        redisCommandExecutor);
+    redisCommandExecutor =
+        LoggingExecutors.newCachedThreadPool("Redis Command", true);
```

Comment:
This is an example of why it is better not to change a bunch of formatting. Just let spotless do the formatting. In this case I think you accidently lost a change to the thread name. Someone else changed it to "GeodeRedisServer-Command-" and I think you lost that and reverted to back to "Redis Command". You should have had a conflict but when you have changed a bunch of formatting it can be hard for you to see things like this.

Reply:
makes sense.  I think I'm going to slowly lobby to see if I can get some things updated in Spotless...